src/components/valet-reservation/ReservationList.jsx

```diff
@@ -22,23 +16,10 @@ const ReservationList = () => {
     })
   }, [time, showNextHour, reservations]);
 
-  const fetchReservations = useCallback(async () => {
-    try {
-      const data = await getReservationList(garage_id, date);
-      dispatch(setReservations(data))
-    } catch (error) {
-      console.log(error);
-    }
-  }, [garage_id, date, dispatch]);
-
-  useEffect(() => {
-    fetchReservations();
-  }, [time, fetchReservations]);
-
   useEffect(() => {
-    const interval = setInterval(() => dispatch(setTime()), 1000 * 60 * 60); //check every hour
-    return () => clearInterval(interval);
-  }, [dispatch]);
+    console.log('checking date in list: ', date);
```

Comment:
These logs should be removed if you're no longer testing things.

Reply:
Still testing some things, but yes I will remove them